src/DigitalSubscriberAppBanner.tsx

```diff
@@ -284,18 +287,34 @@ export const storeIcon = css`
 `;
 
 export const DigitalSubscriberAppBanner: React.FC<Props> = ({
-    onButtonClick,
+    logButtonClickWithBraze,
+    submitComponentEvent,
     brazeMessageProps: { header, body },
 }: Props) => {
     const [showBanner, setShowBanner] = useState(true);
 
     const onCloseClick = (
         evt: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-        buttonId: number,
+        internalButtonId: number,
     ): void => {
         evt.preventDefault();
-        onButtonClick(buttonId);
+
         setShowBanner(false);
+
+        // Braze displays button id from 1, but internal representation is numbered from 0
+        // This means that the Button ID in Braze and Ophan will be the same
+        const externalButtonId = internalButtonId + 1;
+        submitComponentEvent({
+            component: {
+                // TODO: Fix this, the types need updating:
+                componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
```

Comment:
Note that this needs updating. We want to be using `RETENTION_ENGAGEMENT_BANNER`, which is yet to be added as a new component type.